client/components/app.jsx

```diff
@@ -1,25 +1,36 @@
 import React from 'react';
-
+import Title from './title';
+import ProductBoard from './product';
 export default class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      message: null,
-      isLoading: true
+      products: []
```

Comment:
You should load the products data into the `ProductList` component instead of `App`.